src/test/java/com/nurkiewicz/java8/J11_AtomicTest.java

```diff
@@ -57,7 +55,7 @@ public void shouldCountEventsInMultipleThreads() throws Exception {
 		MultiRunner.runMultiThreaded(1000, () -> counter.incBy(1));
 
 		//then
-		await().untilCall(to(counter).longValue(), is(1000L));
+		await().until(() -> assertThat(counter.longValue()).isEqualTo(1000L));
```

Comment:
For people how know Hamcrest probably. I might prefer the syntaxt with AssertJ, because I like AssertJ and maybe because I implemented AssertJ assertions support in Awaitility (so it must be good ;) ).